### PR TITLE
Created function to send company name to backend

### PR DESCRIPTION
**Pull request by zaidokal**
# JIRA Ticket Link

https://skaarkzo-capstone.atlassian.net/browse/SUSTAIN-54

# Summary of Changes

Created function to send company name to backend

### CODE REVIEW

File: sustain/src/app/hooks/EvaluateCompanySearch.ts [hunk 1]
```diff
@@ -7,9 +7,27 @@ export default function EvaluateCompanySearch() {
...
+          method: "POST",
+          headers: {
+            "Content-Type": "application/json",
+          },
+          body: JSON.stringify({ companyName: inputValue }),
+        });
+
+        if (response.ok) {
+          const result = await response.json();
+          console.log("Received from backend:", result);
```
Comment (rakhshankhan) [inline sustain/src/app/hooks/EvaluateCompanySearch.ts:23]: I'm assuming these console.log statements will be removed once the actual implementation for this is added?

Comment (rakhshankhan) [reply sustain/src/app/hooks/EvaluateCompanySearch.ts:23]: Also, in terms of design decisions, I think the logic will start to get pretty complex in these hooks when lots of functionalities are being added. I'm wondering if you wanna research some frontend architecture that makes sure all the processing done on the response or fetching the response is "clean"?

**Review by rakhshankhan — DISMISSED** [review]

File: sustain/src/app/hooks/EvaluateCompanySearch.ts (hunk 1 above)
Comment (rakhshankhan) [reply sustain/src/app/hooks/EvaluateCompanySearch.ts:23]: Also, in terms of design decisions, I think the logic will start to get pretty complex in these hooks when lots of functionalities are being added. I'm wondering if you wanna research some frontend architecture that makes sure all the processing done on the response or fetching the response is "clean"?

Comment (zaidokal) [reply sustain/src/app/hooks/EvaluateCompanySearch.ts:23]: Yes the log statements will be removed, only there now to make sure it is working. 

Will look into other possibilities for doing this.

Comment (zaidokal) [reply sustain/src/app/hooks/EvaluateCompanySearch.ts]: Found using reducer pattern will help keeping everything organized. Have updated the code.

Comment (zaidokal) [reply sustain/src/app/hooks/EvaluateCompanySearch.ts]: Reverted reducer pattern and separated into different files to keep it organized and scalable

**Review by rakhshankhan — APPROVED** [review]

**Review by skhan943 — APPROVED** [review]
LGTM

**Review by arhamansarii — APPROVED** [review]

**Pull request merged**